modules/cloudsql-instance/outputs.tf

```diff
@@ -34,6 +34,11 @@ output "connection_names" {
   }
 }
 
+output "dns_name" {
```

Comment:
I think we should output dns names for all instances too, like in `id`, `ids`; `ip`, `ips` etc.

Reply:
Done (: Also for psc_service_attachment_link